examples/next/compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
```

Comment:
The filename should be `docker-compose.yml`

Reply:
Not anymore
https://docs.docker.com/compose/compose-file/03-compose-file/

> The default path for a Compose file is compose.yaml (preferred) or compose.yml that is placed in the working directory. Compose also supports docker-compose.yaml and docker-compose.yml for backwards compatibility of earlier versions. If both files exist, Compose prefers the canonical compose.yaml.

Reply:
I wasn't aware of this change!